src/BaGetter.Core/Configuration/StatisticsOptions.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+
+namespace BaGetter.Core;
+
+public class StatisticsOptions : IValidatableObject
+{
+    [Required]
+    public bool? EnableStatisticsPage { get; set; } = false;
```

Comment:
Why are these required+nullable+default false? Do they have to be nullable if they have a default anyway? Also what does the Required attribute do, is it in case others construct the options themselves?

Reply:
Indeed [Required] does nothing at all. And the nullable boolen either.